net-mgmt/pfSense-pkg-snmptt/files/usr/local/pkg/snmptt.xml

```diff
@@ -0,0 +1,730 @@
+			<fieldname>translate_varname_oid_format</fieldname>
+			<description>
+				<![CDATA[
+					Configures how the symbolic trap OID will be displayed for $v, $-n, $+n, $-* and $+*.<br/>
+					Set to 1, 2, 3 or 4.  See translate_value_oids options 1,2,3 and 4.<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>1</default_value>
+			<options>
+				<option><name>1</name><value>1</value></option>
+				<option><name>2</name><value>2</value></option>
+				<option><name>3</name><value>3</value></option>
+				<option><name>4</name><value>4</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Translate Integers</fielddescr>
+			<fieldname>translate_integers</fieldname>
+			<description>
+				<![CDATA[
+					Set to 0 to disable converting INTEGER values to enumeration tags as defined in the MIB files.<br/>
+					Set to 1 to enable converting INTEGER values to enumeration tags as defined in the MIB files.<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>1</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Wildcard Expansion Separator</fielddescr>
+			<fieldname>wildcard_expansion_separator</fieldname>
+			<description>
+				<![CDATA[
+					Set what is used to separate variables when wildcards are expanded on the FORMAT / EXEC line.<br/>
+					Defaults to a space.  Value MUST be within quotes. Can contain 1 or more characters.<br/>
+				]]>
+			</description>
+			<default_value>" "</default_value>
+			<type>input</type>
+			<size>50</size>
+			<required>true</required>
+		</field>
+		<field>
+			<fielddescr>Allow Unsafe Regex</fielddescr>
+			<fieldname>allow_unsafe_regex</fieldname>
+			<description>
+				<![CDATA[
+					Set to 1 to allow unsafe REGEX code to be executed.<br/>
+					Set to 0 to prevent unsafe REGEX code from being executed (default).<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>0</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Remove Backslash From Quotes</fielddescr>
+			<fieldname>remove_backslash_from_quotes</fieldname>
+			<description>
+				<![CDATA[
+					Set to 1 to have the backslash (escape) removed from quotes passed from<br/>
+					snmptrapd. For example, \" would be changed to just " Set to 0 to disable.<br/>
+				]]>
+			</description>
+			<type>select</type>
+			<default_value>0</default_value>
+			<options>
+				<option><name>0</name><value>0</value></option>
+				<option><name>1</name><value>1</value></option>
+			</options>
+		</field>
+		<field>
+			<fielddescr>Dynamic Nodes</fielddescr>
+			<fieldname>dynamic_nodes</fieldname>
+			<description>
+				<![CDATA[
+					Set to 1 to have NODES files loaded each time a trap is processed.<br/>
```

Comment:
Put short descriptions of these behaviors in the "name" part of the options list.